### ISSUE TITLE
Log says rip was successful but no rip exists (UHD)

### ISSUE THREAD

**Issue by XJonnyP** [rix1337/docker-ripper#73]
I guess the simple question is, does this support 4K UHD disks? 

I just set up the docker container on a machine running UNRAID. I insert a disc and it gets picked up and after only a few seconds later it says the job is done and ejects the disk.

```06.07.2021 13:30:36 : Starting Ripper. Optical Discs will be detected and ripped within 60 seconds.
06.07.2021 13:32:57 : BluRay detected: Saving MKV
renamed '/out/Ripper/BluRay/BABY_DRIVER' -> '/out/Ripper/BluRay/finished/'
06.07.2021 13:33:09 : Done! Ejecting Disk
```

This is a 4K UHD disk. My blu ray drive has firmware which can read these (previously used on a windows machine). I tried with a standard bluray and it has been on `Saving MKV` for about 10 minutes which I assume means it's actually working.

**Comment by XJonnyP** [comment]
I _suspect_ this may have something to do with the encryption key for the disk. I tried an makemkv only container and got an 'unknown volume key' error for this disk.

Tried another UHD disk in the docker-ripper container and it had the same behavior (indicated success after a few seconds) but this one seems to be running fine through the makemkv decryption process. 

At this point not sure if this is a bug per se or just a lack of having the language in logging to indicate the decryption failed (if in fact this container is even configured to handle decryption of UHD)
